### ISSUE TITLE
Have you used your term (1) in loss?

### ISSUE THREAD

**Issue by jsrdcht** [zaixizhang/CBD#3]
i.e., **I(X, Z)**
It seems that term (1)  wasn't used in training.

**Comment by zaixizhang** [comment]
Hi,
Thanks for your interest in our work! In the paper, we show that optimizing I(X,Z) equals applying L2 norm regularization on the embedding vector z. In ablation studies, we find such a term does not influence the results very much. It can be simply integrated into the training by applying regularization or weight decay. 

Bests,
